### ISSUE TITLE
TypeError: Cannot read property 'inconsiderate' of undefined

### ISSUE THREAD

**Issue by blakeembrey** [wooorm/atom-linter-alex#1]
```
TypeError: Cannot read property 'inconsiderate' of undefined
    at check (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/lib/equality.js:172:28)
    at test (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/lib/equality.js:325:18)
    at /Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/lib/equality.js:525:26
    at one (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:96:22)
    at /Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:83:29
    at forwards (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:24:13)
    at all (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:82:16)
    at one (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:100:20)
    at /Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:83:29
    at forwards (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:24:13)
    at all (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:82:16)
    at one (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:100:20)
    at visit (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:106:5)
    at /Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/lib/equality.js:509:9
    at one (/Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:96:22)
    at /Users/blakeembrey/.atom/packages/linter-alex/node_modules/alex/node_modules/retext-equality/node_modules/unist-util-visit/index.js:83:29
```

Happening every keypress now. Probably related to incorrectly checking the prototype as it started once I typed `constructor`.

**Comment by blakeembrey** [comment]
Looks like it's https://github.com/wooorm/retext-equality/blob/master/lib/equality.js#L510-L522, seems you tried to patch it but it doesn't look like it does anything since you override it all again right afterwards.

**Comment by wooorm** [comment]
Oh I think I know where this is, it probably has to do with something like 'tostring' or 'constructor' in your text? I'll update soon :)

**Comment by wooorm** [comment]
I fixed it and will push an update in another hour

**Comment by wooorm** [comment]
Whoops, this was closed by wooorm/retext-equality@69f96517e8a00387f9ba832dc9b8f51c6f1427e3!

**Comment by wooorm** [comment]
You’ll have to reinstall the linter though :(
